### ISSUE TITLE
Error on startup [latest version only]

### ISSUE THREAD

**Issue by RikoDEV** [bensku/Skript#181]
Logs:

```
 [08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! 
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! [Skript] Severe Error:
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! Exception in thread Skript offline player resolver thread (fetches UUIDs from the minecraft servers)
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! 
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! If you're developing an add-on for Skript this likely means that you have done something wrong.
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! If you're a server admin however please go to https://github.com/bensku/Skript/issues/
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! and check whether this error has already been reported.
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! If not please create a new ticket with a meaningful title, copy & paste this whole error into it (or use paste service),
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! and describe what you did before it happened and/or what you think caused the error.
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! If you think that it's a trigger that's causing the error please post the trigger as well.
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! By following this guide fixing the error should be easy and done fast.
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! 
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! Stack trace:
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! java.lang.NullPointerException
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#!     at ch.njol.skript.bukkitutil.UnresolvedOfflinePlayer$1.run(UnresolvedOfflinePlayer.java:59)
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#!     at java.lang.Thread.run(Thread.java:745)
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! 
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! Version Information:
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#!   Skript: 2.2-dev19
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#!   Bukkit: 1.10.2-R0.1-SNAPSHOT
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#!   Minecraft: 1.10.2
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#!   Java: 1.8.0_91 (Java HotSpot(TM) 64-Bit Server VM 25.91-b14)
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#!   OS: Linux amd64 2.6.32-042stab108.8
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! 
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! Running CraftBukkit: false
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! 
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! Current node: null
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! Current item: null
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! 
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! Thread: Skript offline player resolver thread (fetches UUIDs from the minecraft servers)
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! 
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! End of Error.
[08:08:36] [Skript offline player resolver thread (fetches UUIDs from the minecraft servers)/ERROR]: #!#! 
```

**Comment by bensku** [comment]
Very strange. I wonder if changing the compiler broke something...

**Comment by RikoDEV** [comment]
@bensku next error ;v
```
[13:21:57] [Server thread/ERROR]: Could not pass event BlockPlaceEvent to Skript v2.2-dev19
java.lang.NoSuchMethodError: org.bukkit.Bukkit.getOnlinePlayers()[Lorg/bukkit/entity/Player;
	at com.w00tmast3r.skquery.elements.effects.EffParticle.execute(EffParticle.java:33) ~[?:?]
	at ch.njol.skript.lang.Effect.run(Effect.java:54) ~[?:?]
	at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:65) ~[?:?]
	at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:93) ~[?:?]
	at ch.njol.skript.lang.Trigger.execute(Trigger.java:53) ~[?:?]
	at ch.njol.skript.SkriptEventHandler.check(SkriptEventHandler.java:152) ~[?:?]
	at ch.njol.skript.SkriptEventHandler$1.execute(SkriptEventHandler.java:112) ~[?:?]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[patched_1.10.2.jar:git-Paper-845]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.10.2.jar:git-Paper-845]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:517) [patched_1.10.2.jar:git-Paper-845]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:502) [patched_1.10.2.jar:git-Paper-845]
	at org.bukkit.craftbukkit.v1_10_R1.event.CraftEventFactory.callBlockPlaceEvent(CraftEventFactory.java:146) [patched_1.10.2.jar:git-Paper-845]
	at net.minecraft.server.v1_10_R1.ItemStack.placeItem(ItemStack.java:162) [patched_1.10.2.jar:git-Paper-845]
	at net.minecraft.server.v1_10_R1.PlayerInteractManager.a(PlayerInteractManager.java:534) [patched_1.10.2.jar:git-Paper-845]
	at net.minecraft.server.v1_10_R1.PlayerConnection.a(PlayerConnection.java:891) [patched_1.10.2.jar:git-Paper-845]
	at net.minecraft.server.v1_10_R1.PacketPlayInUseItem.a(SourceFile:55) [patched_1.10.2.jar:git-Paper-845]
	at net.minecraft.server.v1_10_R1.PacketPlayInUseItem.a(SourceFile:11) [patched_1.10.2.jar:git-Paper-845]
	at net.minecraft.server.v1_10_R1.PlayerConnectionUtils$1.run(SourceFile:13) [patched_1.10.2.jar:git-Paper-845]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_91]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_91]
	at net.minecraft.server.v1_10_R1.SystemUtils.a(SourceFile:45) [patched_1.10.2.jar:git-Paper-845]
	at net.minecraft.server.v1_10_R1.MinecraftServer.D(MinecraftServer.java:791) [patched_1.10.2.jar:git-Paper-845]
	at net.minecraft.server.v1_10_R1.DedicatedServer.D(DedicatedServer.java:404) [patched_1.10.2.jar:git-Paper-845]
	at net.minecraft.server.v1_10_R1.MinecraftServer.C(MinecraftServer.java:725) [patched_1.10.2.jar:git-Paper-845]
	at net.minecraft.server.v1_10_R1.MinecraftServer.run(MinecraftServer.java:624) [patched_1.10.2.jar:git-Paper-845]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_91]
```

**Comment by bensku** [comment]
SkQuery error, won't fix.

**Comment by Hexivoid** [comment]
Have the same issue. Still isn't fixed in the latest version of SkQuery.

**Comment by TheBentoBox** [comment]
You have to use a 1.9 version of SkQuery to fix that. Either run the 1.8 jar through the Spigot SpecialSource jar fixer that they released with 1.9, or use the 1.9 version of SkQuery from [Gatt's site](https://umbaska.gatt.space/).
